04-spatial-operations.qmd

```diff
@@ -145,6 +145,15 @@ canterbury_southland_height = nz_height[sel]
 canterbury_southland_height
 ```
 
+or, alternatively, we can use the `.isin` function to filter the regions and use `.overlay` to calculate the pairwise intersection between the `canterbury_southland` GeoDataFrame (with two rows for Canterbury and Southland, respectively) and the `nz_height` GeoDataFrame: 
+
+```{python}
+canterbury_southland = nz[nz['Name'].isin(['Canterbury', 'Southland'])]
+canterbury_southland
+nz_height.overlay(canterbury_southland)
```

Comment:
If you skipped overlay on purpose (e.g. because you want to introduce it later) please disregard this suggestion. I think .isin is better than chaining multiple = operations though.

Reply:
Going to merge `main` so I can 'Review in codespace'

![image](https://github.com/geocompx/geocompy/assets/1825120/2b31f1d5-ca2e-4cc3-8303-e23cf49c2980)